go.mod

```
@@ -1,6 +1,6 @@
 module github.com/pivotal-cf/brokerapi/v10
 
-go 1.19
+go 1.21
```

```suggestion
go 1.20
```

As suggested by Marcela, since we want to convey that this version is compatible with go 1.20 & 1.21, it makes sense to specify `go 1.20` in go.mod instead of `go 1.21`